study-builder/studies/pancan/emails/consent_completed.en.html

```diff
@@ -1,16 +1,16 @@
 #set( $text = {
     "subject": "Thank you for providing your consent",
-    "logo_alt": "Pancan Project logo",
+    "logo_alt": "Count Me In Project logo",
     "salutation_self": "Dear Participant,",
     "salutation_parent": "Dear Parent,",
     "thank_you": "Thank you for joining Count Me In and for giving us your consent for research.",
     "what_to_expect": "We will next mail you a kit with instructions on how to provide a saliva sample.",
-    "what_to_expect_sample_request": "In some cases, we may also contact you about providing a blood sample (1 tube or 2 teaspoons per sample), which can be drawn at your physician's office, local clinic, or nearby lab facility. If applicable, we will provide detailed instructions on how to do this. We may obtain copies of {{~#if proxy_firstName~}}your child's{{~else~}}your{{~/if~}} medical records and, after reviewing them, may also request a small portion of stored tumor sample from the hospital where it is kept. Once we have analyzed {{~#if proxy_firstName~}}your child's{{~else~}}your{{~/if~}} requested samples, we will return any unused tumor samples to the pathology department that sent them to us.",
-    "next_steps": "If you have not yet provided information about where {{~#if proxy_firstName~}}your child has{{~else~}}you have{{~/if~}} received care for your cancer(s), you can complete the process here:",
```

Comment:
Using Handler's whitespace control (e.g. `{{~#if .. ~}}` is not ideal because it cuts out the whitespace between the words, resulting in this:

![Screen Shot 2021-07-22 at 6 11 56 PM](https://user-images.githubusercontent.com/5713833/126719823-ed9eb1fe-4c0c-4b06-9a96-82281d9f4425.png)